src/scripts/main.js

```diff
@@ -21,7 +20,19 @@ const food = {
 const tree = document.querySelector('#tree');
 
 function createTree(element, data) {
-  // WRITE YOUR CODE HERE
+  const ul = document.createElement('ul');
+
+  for (let key in data) {
+    const li = document.createElement('li');
+    li.textContent = key;
+    ul.appendChild(li);
+
+    if (typeof data[key] === 'object' && Object.keys(data[key]).length > 0) {
+      li.appendChild(createTree(li, data[key]));
+    }
+  }
+
+  return ul;
 }
```

Comment:
You are using a recursive function to create nested lists. This is a good approach, but you should be aware that recursion can lead to performance issues if the depth of the recursion is very large. In this case, it's not a problem because the depth of the `food` object is not very large.